lib/product_list.dart

```diff
@@ -6,19 +6,14 @@ import 'product_details.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 import 'package:sentry/sentry.dart';
 
-
 class ItemsList extends StatefulWidget {
   _ItemListState createState() => _ItemListState();
 }
 
 class _ItemListState extends State<ItemsList> {
-  final String _uri = 'https://application-monitoring-flask-dot-sales-engineering-sf.appspot.com/products';
+  final String _uri =
+      'https://application-monitoring-flask-dot-sales-engineering-sf.appspot.com/products';
   late Future<ResponseData> shopItems;
-  final transaction = Sentry.startTransaction(
```

Comment:
We don't need this anymore, a transaction will be created automatically for screen loading.